data/json/items/containers.json

```diff
@@ -273,10 +273,19 @@
     "qualities": [ [ "BOIL", 1 ] ]
   },
   {
-    "id": "bottle_plastic",
+    "id": "bottle_plastic_sealed",
     "type": "GENERIC",
     "category": "container",
     "name": { "str": "plastic bottle" },
+    "copy-from": "bottle_plastic",
+    "description": "A watertight plastic bottle, holds 500 ml of liquid. Preserves content from rot until opened.",
+    "pocket_data": [ { "pocket_type": "CONTAINER", "spoil_multiplier": 0.5, "sealed_data": { "spoil_multiplier": 0.0 } } ]
+  },
+  {
+    "id": "bottle_plastic",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "opened plastic bottle" },
```

Comment:
It is not necessarily opened, just not able to preserve content from rot. And it can still be confusing if the player goes into a store and finds out that every bottle is "opened". I think it would be better if we named `bottle_plastic_sealed` something that reflects that it is originally sealed to preserve its content while not hinting at whether it's currently sealed or not, but I can't think of anything good right now.

Reply:
I.. don't have a good name for that. Any ideas

Reply:
'Unsealed' sounds too much like the 'unsealed tin can', but certainly better than 'opened'. "Un-factory-sealed" is a mouthful on the other hand - any ideas?